aten/src/ATen/native/CPUBlas.cpp

```diff
@@ -324,6 +324,46 @@ void gemm(
      return;
    }
 #endif
+
+#if AT_BUILD_WITH_BLAS()
+  if (use_blas_gemm(transa, transb, m, n, k, lda, ldb, ldc)) {
+    int m_ = m, n_ = n, k_ = k, lda_ = lda, ldb_ = ldb, ldc_ = ldc;
+    float alpha_ = alpha, beta_ = beta;
+    int a_size = (transa != TransposeType::NoTranspose?m_:k_) * lda;
+    std::vector<float> float_a(a, a + a_size);
+    int b_size = (transb != TransposeType::NoTranspose?k_:n_) * ldb;
+    std::vector<float> float_b(b, b + b_size);
+    int c_size = n_ * ldc;
```

Comment:
Matrix size does not change, when it's transposed, does it?
```suggestion
    int a_size =  lda * k;
    std::vector<float> float_a(a, a + a_size);
    int b_size = ldb * n;
    std::vector<float> float_b(b, b + b_size);
    int c_size =  ldc * n;
```